src/main/java/com/conveyal/gtfs/model/Pattern.java

```diff
@@ -114,7 +113,7 @@ public Pattern () {}
     public static class Loader extends Entity.Loader<Pattern> {
 
         public Loader(GTFSFeed feed) {
-            super(feed, Table.proprietaryFilePrefix + "patterns");
+            super(feed, Table.PROPRIETARY_FILE_PREFIX + "patterns");
```

Comment:
This and the ctor at line 144, and also the code from `getTableFileName`: extract logic to build the table file name without the extension. Maybe add an overload to `getTableName` to accept the table prefix?